actor-typed-tests/src/test/scala/docs/org/apache/pekko/typed/InteractionPatternsSpec.scala

```diff
@@ -359,13 +367,6 @@ class InteractionPatternsSpec extends ScalaTestWithActorTestKit with AnyWordSpec
   }
 
   "contain a sample for per session child" in {
-    // #per-session-child
-    // dummy data types just for this sample
-    case class Keys()
```

Comment:
So this change (which involved moving `Keys` and `Wallet()` outside of a class) is due to a new error that Scala 3.3 reports. This error is due to the fact that checking for structural equality of classes nested within classes is ambiguous (in Scala 2 this was already a warning).

Solution is quite easy, just move the classes outside of an inner class. Note that this is just a test/sample so its not going to break anything.